srcpkgs/vifm/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'vifm'
 pkgname=vifm
-version=0.12
+version=0.13
 revision=1
 build_style=gnu-configure
 configure_args="--without-gtk"
-hostmakedepends="mdocml"
+hostmakedepends="mdocml perl coreutils"
```

Comment:
this makes no sense, coreutils are installed by default.

Reply:
A fix could look like adding this at the top of `src/sort.c`:
```c
#ifdef HAVE_STRVERSCMP_FUNC
#    undef HAVE_STRVERSCMP_FUNC
#endif
```
(`#undef` alone might also work, but just in case.)
This will always use custom implementation of `strverscmp()` (Windows doesn't have it and BSD-systems probably too).  The tests validate custom implementation and that behaviour is consistent across various systems.